iron-form.html

```diff
@@ -288,7 +295,8 @@
         // were already added as a custom element, they don't need
         // to be re-added.
         if (!this._useValue(el) ||
-            (el.hasAttribute('is') && json[el.name])) {
+            (el.hasAttribute('is') && json[el.name]) || 
```

Comment:
If it was already added then `json[el.name]` should === `el.value`, no? And it that case we can skip it?